jerry-ext/include/jerryscript-ext/section.impl.h

```diff
@@ -0,0 +1,56 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
```

Comment:
The name section.impl.h does not refer to modules. It is clear that It is an internal header and should not be used, but it is still part of the public API.